app/scripts/modules/core/src/search/global/GlobalSearchResults.tsx

```diff
@@ -0,0 +1,79 @@
+import * as React from 'react';
+import { UISref } from '@uirouter/react';
+
+import { SETTINGS } from 'core/config/settings';
+import { ISearchResult } from 'core/search/search.service';
+import { ISearchResultSet } from 'core/search/infrastructure/infrastructureSearch.service';
+import { SearchResult } from 'core/search/infrastructure/SearchResult';
+
+export interface IGlobalSearchResultsProps {
+  categories: ISearchResultSet[];
+  query: string;
+  onItemKeyDown: (event: React.KeyboardEvent<HTMLAnchorElement>) => any;
+  onResultClick: (result: ISearchResult) => any;
+  onSeeMoreClick: Function;
```

Comment:
Why is this a `Function` but the others are `(...) => any;`?

Reply:
er. yeah. this was me trying to typescript and getting better at it incrementally. and then not going back and revising things after i got better at it.